neqo-bin/benches/main.rs

```diff
@@ -20,7 +20,8 @@ fn transfer(c: &mut Criterion) {
     neqo_common::log::init(Some(log::LevelFilter::Off));
     neqo_crypto::init_db(PathBuf::from_str("../test-fixture/db").unwrap()).unwrap();
 
-    let done_sender = spawn_server();
+    let port = 49151 + (rand::random::<u16>() % 16384);
```

Comment:
Should we use neqo-crypto here? It doesn't seem to have a clean way to return randomized types, just `u8`?